source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRTextView.mm

```diff
@@ -44,6 +44,8 @@ - (void)configWithSharedModel:(ACOBaseCardElement *)element
     self.delegate = self;
     self.id = [NSString stringWithCString:inputBlck->GetId().c_str()
                                  encoding:NSUTF8StringEncoding];
+    self.accessibilityIdentifier = self.id;
+    self.accessibilityLabel = self.id;
```

Comment:
i wouldn't change the default label of self.assesiblityLabel to self.id. UIView objects get their asseiblityLabel automatically.

Reply:
what value do they get it from?